backend/middleware/errorMiddleware.ts

```diff
@@ -21,7 +22,7 @@ const errorHandler = (
   res.status(statusCode)
   res.json({
     message: err.message,
-    stack: process.env.NODE_ENV === 'production' ? null : err.stack
+    stack: config.productionApplication ? null : err.stack
```

Comment:
Remember: do not use the value coming from `NODE_ENV` to implement logic. Do the same we did with the `SERVE_REACT_APP` var for this too.

Reply:
changed to "shouldServeReactApp" as well!